package.json

```diff
@@ -65,13 +65,13 @@
     "react-responsive": "^8.2.0",
     "react-table": "^7.7.0",
     "react-transition-group": "^4.4.2",
-    "tabbable": "^4.0.0",
+    "tabbable": "^5.3.3",
```

Comment:
[curious/confirmation] Were any code changes necessary when moving to v5 of `tabbable`? According to its CHANGELOG, it looks like the default export was changed to a named export (e.g., `import { tabbable } from 'tabbable';`). This only seems to affect the deprecated `Modal` component. Just wanted to make sure this upgrade isn't breaking any of `Modal`'s behavior.